2020-04-07-Charity-Shop-Exchange.md

```diff
@@ -0,0 +1,62 @@
+
+**Can you describe your project?**
+
+Charity Shop Exchange offers a subscription service of books & DVDs. People need new things to read and watch - especially when feeling isolated - and our subscription service facilitates this in an economical way. Subscribers receive a (weekly, bi-weekly or monthly)  selection of books and DVDs which have been sourced from local charity shops.
+
+**How did your team come together?**
+
+All of us met through *Slack*, or *Twitter* while trying to discover a way we could help out during this crisis. Kelvin had taken the steps to reach out to us individually, add us to a Slack channel so we could discuss the concept, and from there we got right down to it!
+
+**Can you tell us about your team?**
+
+Kelvin Lockwood  hails from Bournemouth, UK
+I am from Sioux Falls, South Dakota
+Matt Doyle lives in Sydney Australia
+Dicky Jiang is based in Bandung Area, West Java, Indonesia
+
+**What skills do they bring?**
+
+Kelvin Lockwood - Product Ownership and Product Developer
+I am a Full Stack Web Developer, HTML, CSS, JS, and React/Node
+Matt Doyle - Full Stack Web Development: HTML, CSS, JS, and React/PHP/WP
+Dicky Jiang - UIX Designer: Photoshop, Photography, Graphic Design, CSS, HTML, and Web Design
+
+**How was the experience of working remotely?**
+
+Could not have been better. Through the use of Github Projects, Google Documents, and Milanote we were able to work through open issues and tasks. Additionally, being that we were all in different time zones, someone was waking up as soon as the other was going to sleep, so we were nearly making progress around the clock.
+
+**How does it feel to be collaborating on a project at a time like this?**
+
+A great benefit, we’ve all shared with each other the additional steps we’ve taken to remain social and cope with the challenges we’ve faced. Whether it’s on a zoom-call or just in Slack, the frequent conversation and working towards a goal is therapeutic through and through. 
+
+**What tools, made available by the  hackathon, are you using?**
+
+At this time we’ve not had a need for a Back end component on our project, but would like to thank FaunaDB for offering their database services to the hackathon, I’ll be sure to use them in an upcoming project!
+
+**Is there anything else you’d like to say?**
+
+We would just like to thank everyone involved in this collaborative event and for allowing us to share what we’ve built!
+
+Blake *A.* Prouty
+
+For more information on Charity Shop Exchange:
+
+Website: [Charity Shop Exchange](https://charityshopexchange.com)
+Source code: [CSE code](https://github.com/Charity-Shop-Exchange/Charity-Shop-Exchange)
+Country: United Kingdom, USA, Australia and Indonesia
+
```

Comment:
@johnny-newton I'd recommend pulling this second blog post out, making sure it's in the `_posts` folder, and submitting it as a separate PR on a separate branch.  We still have to make a couple fixes to the first blog post wrt image URLs.

If you need help with this, I can try, but I don't know what you're currently using to author the posts and thus manage Git commits.  In my experience, "undoing" situations like this requires a [fundamental understanding](https://learngitbranching.js.org/) of how Git works.